.github/workflows/merge-release-tag.yml

```
@@ -0,0 +1,57 @@
+name: Merge Release Tag
+
+on:
+  push:
+    tags:
+      - 'mongosh@[0-9]+.[0-9]+.*'
```

We can still enforce the 3 components:

```suggestion
      - 'mongosh@[0-9]+\.[0-9]+\.[0-9]+.*'
```

Though I'm not sure how strict we have to be here - I don't really expect we'll be pushing malformed `mongosh@...` tags

---

not sure escaping is right, this isn't actually regex: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet